### ISSUE TITLE
enableAdvertisingTracking

### ISSUE THREAD

**Issue by prundianudaniel** [claimsforce-gmbh/flutter-segment#25]
If i do this (**iOS**): 

```
await Segment.setContext({
    'device': {
    'token': currentToken,
    }
});
```

my advertismentIdentifier in the device config in segments debugger is gone. 
Do i do something wrong here?

**Expected**:

"device": {
      "adTrackingEnabled": true,
      "advertisingId": "***",
      "id": "***",
      "manufacturer": "Apple",
      "model": "x86_64",
      "name": "iPhone",
      "type": "ios",
      "token": "mytoken"
    },

**Result**:

"device": {
      "token": null,
      "type": "ios"
    },

**Comment by MaiKaY** [comment]
Hi @prundianudaniel,
which version are you using?
Have you check https://github.com/claimsforce-gmbh/flutter-segment/issues/14? Maybe it's related to your problem.

**Comment by prundianudaniel** [comment]
i use 2.1.0 and yes @MaiKaY thats exactly my problem

**Comment by MaiKaY** [comment]
I can reproduce that as well.

@vinicius-animo maybe it's solved for Android but not for iOS?

**Comment by MaiKaY** [comment]
I opened a new issue but this time for the Segment iOS library: https://github.com/segmentio/analytics-ios/issues/907

**Comment by MaiKaY** [comment]
Hi @prundianudaniel,
should be fixed with our latest version [`2.1.1`](https://pub.dev/packages/flutter_segment).

Feel free to re-open in case it's not fixed.
